### ISSUE TITLE
3D not rendering with styled 5.0.4

### ISSUE THREAD

**Issue by garybrain** [highcharts/highcharts#6018]
#### Expected behaviour

Chart to be rendered in 3D

#### Actual behaviour

Chart showing in 2D due to JS error.

highcharts-3d.js:26 Uncaught TypeError: Cannot read property 'style' of undefined(…)(anonymous function) 
@ highcharts-3d.js:26(anonymous function) 
@ highcharts-3d.js:28(anonymous function) 
@ highcharts-3d.js:8(anonymous function) 
@ highcharts-3d.js:8

#### Live demo with steps to reproduce

Styled. ( error ).
using : https://code.highcharts.com/js/highcharts-3d.js
http://jsfiddle.net/cyn0xkk8/

Using following 3d.js works.
using : https://code.highcharts.com/highcharts-3d.js
http://jsfiddle.net/cyn0xkk8/1/

#### Affected browser(s)
All browsers

**Comment by pawelfus** [comment]
Thank you for reporting!

This is a regression from 62003e64411795f532b77281f041bc4b2a433142

**Workaround:**
Set `defs.style.textContent` before loading `highcharts-3d.js` file and add side/top CSS, Demo: http://jsfiddle.net/cyn0xkk8/2/

Snippet:

```html
<script src="https://code.highcharts.com/js/highcharts.src.js"></script>
<script>
  Highcharts.setOptions({
    defs: {
      style: {
        textContent: ''
      }
    }
  });
</script>
<script src="https://code.highcharts.com/js/highcharts-3d.src.js"></script>

```

CSS:
```css
.highcharts-3d-top {
  filter: url(#highcharts-brighter)
}

.highcharts-3d-side {
  filter: url(#highcharts-darker)
}
```

**Comment by garybrain** [comment]
Many thanks

G,

**Comment by Dams591** [comment]
Hey guys 
Was wondering if this will be fixed/pushed out with 5.0.5 and if so if you have a release date. 
Cheers.
